src/templates/QuickstartDetails.js

```diff
@@ -49,6 +50,16 @@ const QuickstartDetails = ({ data, location }) => {
   const pack = data.quickstarts;
   const packUrl = pack.packUrl || QUICKSTARTS_REPO;
   const tessen = useTessen();
+  const breadcrumbSegments = [
```

Comment:
Totally a style thing (zero pressure to change): I'm curious why you opted for this name?

***breadcrumb***Segments / breadcrumb***Segments*** 

It seems like it has a higher level of specificity than we need. Are there _other_ types of segments? Are there segments for _other_ things? We might be able to get away with less verbosity: something like `breadcrumbs` might be easier to understand.

Or perhaps we don't want to misrepresent what the variable represents! I could be convinced either way...just pondering 🤔

Reply:
🤔 no strong opinion here, other than i do like the shorter `breadcrumbs`. made that change.